skxray/core/image.py

```diff
@@ -97,3 +97,58 @@ def _corr_ax1(input_image):
                                          mode='full')/norm_mask) / 2
              for v in input_image]
     return np.histogram(est_by_row, bins=np.arange(0, dim + 1))
+
+def hist_make(src_data,
+              num_bins,
+              pd_function=False):
+    """
+    This function evaluates the histogram of the source data set and returns
+    bin data consisting of both bin edges and bin averages. This tool is
+    primarily geared for plotting histograms for visual analysis and
+    comparison.
+
+    Parameters
+    ----------
+    src_data : ndarray
+        Can be JxK or IxJxK
+        Specifies the source data set from which you want to evaluate the
+        histogram.
+    num_bins : int
+        Specify the number of bins to include in the histogram as an integer.
+
+    pd_function : bool, optional
```

Comment:
noted. Thanks, didn't think of that (obviously).
Changed to pd_convert. actually, would changing this to "normalize" be more easily understandable?